### PR TITLE
task/FP-1819: Create submitter modal - ui updates

### PR DESCRIPTION
**Pull request by wesleyboar**
## ⚠️ To Do

- [ ] Update markup/styles for new `{{r.reg_status}}` data (2592d90).

## Overview

- Cleanup markup of submitter modal.
- Improve the UI just a little.

## Related

- improves #56

## Changes

- borrowed `c-form` from https://github.com/TACC/Core-Styles
- tweaked modal styles (use portal font, drop title margin)
- cleaned up the markup and use `c-form`
- added client-side validation ([reference](https://github.com/TACC/Core-CMS-Custom/pull/56#discussion_r1054533638))
- loaded new stylesheet
- removed now-unnecessary `&nbsp;` from table

## Testing

1. Open modal.
2. Review UI is improved since #56.

### Table

| Before | After |
| - | - |
| Before (Table) | After (Table) |

### Modal

| Before | After |
| - | - |
| Before (Modal) | After (Modal) |

| Mobile |
| - |
| Modal Mobile |

### CODE REVIEW

File: apcd-cms/src/apps/admin_regis_table/templates/create_submitter_modal.html [hunk 1]
```diff
@@ -1,70 +1,70 @@
...
+              {% if file_type != None %}
+              <dd class="c-data-list__value">{{file_type}}</dd>
+              {% endif %}
+              {% endfor %}
+            </dl>
+          </article>
         </div>
+
+        <form method="POST" action="" class="c-form">
+          <h4>New Data</h4>
```
Comment (wesleyboar) [inline apcd-cms/src/apps/admin_regis_table/templates/create_submitter_modal.html:43]:
```suggestion
          <h4>Submitter Data</h4>
```

**Review by wesleyboar — COMMENTED** [review]
Wes, [2592d90](https://github.com/TACC/Core-CMS-Custom/pull/58/commits/2592d90bdfc98e54f57a06a32636d8bc9f740316) provided new data. Check it out, cuz you probably need to style it.

**Review by edmondsgarrett — APPROVED** [review]
Looks great! Approved.

**Comment by wesleyboar** [comment]
I do worry about styling for `{{r.reg_status}}`.

**Pull request merged**